common/button/index.json-ld

```diff
@@ -0,0 +1,34 @@
+{
+	"@context": {
+		"@version": 2.0,
+		"dct": "http://purl.org/dc/terms/",
+		"title": { "@id": "dct:title", "@container": "@language" },
+		"description": { "@id": "dct:description", "@container": "@language" },
+		"modified": "dct:modified"
+	},
+	"title": {
+		"en": "Buttons",
+		"fr": "Boutons"
+	},
+	"description": {
+		"en": "Buttons page including working examples to test how various button styles appear and function",
+		"fr": "Page des boutons comprenant des exemples de travail pour tester l'apparence et le fonctionnement de divers styles de boutons."
+	},
+	"modified": "2023-06-09",
+	"componentName": "button",
+	"status": "stable",
+	"pages": {
+		"docs": [
```

Comment:
Use examples instead of "docs"

```suggestion
		"examples": [
```

Reply:
Changes made in the commit "Updated working examples".